tools/api-markdown-documenter/.prettierignore

```diff
@@ -1,22 +0,0 @@
-# Generated by npm
```

Comment:
I realized this was unused. This package references the ignore file at the repo root.